app/src/main/java/com/android/halanvendordiscovery/data/details/mapper/MerchantRemoteMapper.kt

```diff
@@ -0,0 +1,17 @@
+package com.android.halanvendordiscovery.data.details.mapper
+
+import com.android.halanvendordiscovery.data.details.model.response.MerchantRemoteModel
+import com.android.halanvendordiscovery.domain.details.model.MerchantDomainModel
+
+object MerchantRemoteMapper{
```

Comment:
MerchantRemoteDomainMapper is more suitable, right?

Reply:
Done 🚀🚀🚀